modules/openapi-generator/src/main/resources/kotlin-multiplatform-client/common/main/infrastructure/ApiClientBase.kt.mustache

```diff
@@ -64,42 +64,42 @@ public abstract class ApiClientBase {
     }
     {{#hasAuthMethods}}
 
-    protected fun HttpRequestBuilder.addAuthentication(apiKeyAuths: List<String>, basicAuths: List<String>, bearerAuths: List<String>, oAuths: List<String>) {
-        for (name in apiKeyAuths) {
-            val auth = apiKeyAuth[name] ?: throw IllegalStateException("ApiKeyAuth \"$name\" was configured, but not found")
```

Comment:
These had the pretty good reason to error on invalid schemas, so I'd like to keep them